### DISCUSSION TITLE
Thoughts on power usage reported by PJ-1203A power clamp

### DISCUSSION THREAD

**Discussion by dmshimself** [Koenkk/zigbee2mqtt#23975] [Devices]
I've been puzzled for a little while as to why my PJ-1203A power clamp reports power usage that is quite different to usage reported by my solar inverter.  They are a factor of about 2 out.

So if I switch everything off and just have a single device with well published power usage powered on, a heat pump that just has fans on, the clamp reports power usage at (say) 200W, but the inverter reports about 400W.  I've put the heat pump into various states, let the usage settle and the difference in power reported is always the same, about twice on the inverter compared with the clamp.  The power factor reported by the clamp is always about 0.85 and doesn't change very much at all.  If I power the heat pump off, power reported from the clamp drops to 0 and about 10W on the inverter.  The inverter always uses a bit of power.

After some testing and looking at possibly some calibration issues, I realised that the different was always a factor of 2.09 and that number rang a bell as it's my voltage (232) / US voltage (110). The voltage reported on the clamp in MQTT is 234V, so pretty close.

This magic number might just be a co-incidence, but I wondered if anyone else has seen similar issues with power reported not matching what they expect with this device?

**Comment by fejib** [comment]
Bonjour, j'ai le même problème: la valeur de l'onduleur est env le double de celle des pinces ampermétriques. Avez-vous trouvé le problème/ solution?

**Comment by dmshimself** [comment]
I still have the same problem, so I just used a template sensor in Home Assistant to correct this.  Something like:

{{ max((232/110) * (float(states.sensor.ct_clamp_power_a.state) ),0) }}

**Reply by fejib** [reply]
It fluctuates for me from time to time, but with this we're getting closer to the value. Thank you for your help.
